### ISSUE TITLE
[Bug] Map not shown

### ISSUE THREAD

**Issue by AdielCohen** [visgl/react-google-maps#511]
### Description

I use this library and it works great for me at home on Windows11 with Chrome or Edge and even on the SAFARI browser on iPhones.
But the college I attend has a Mac computer and when I enter the site through Safari it does not show me the map.
But, If I use Google Map in the Safari browser it does show the map.
I tried to download Chrome to the computer and the map did work, This problem happen only when I use Safari at this computer.
There is no any error in Devtools.
Why this is happen and how can I fix that?

### Steps to Reproduce

1. Open Safari with the map

### Environment

- Library version: ^1.1.0
- Google maps version: weekly
- Browser and Version: Safari version 13.1.2 (13609.3.5.1.5)
- OS: Mac

**Comment by usefulthink** [comment]
Well - to be fair - Safari 13 is over 4 years old already and, according to the [caniuse usage-table](https://caniuse.com/usage-table), has a 0.07% market share. 

Officially, the [browser support for the Maps JavaScript API](https://developers.google.com/maps/documentation/javascript/browsersupport) only covers the last two major versions of all browsers (in case of Safari that's versions 16 and 17), although in reality it's usually more than that, but no guarantees given.

Without knowing what it is that Safari 13 has a problem with, there's not really something I can recommend. Do you know if the Maps JavaScript API in a current version still works there? How about a current React version?

If both are the case, there might be a way to build a version of this library that also works there. We don't have anything too exotic going on in our codebase, so there could be some set of polyfills or compiler settings to enable support. 

And while that's not something we will put on our roadmap, feel free to give it a try. And if there's a simple thing we can do that doesn't slow down the library for modern browsers or increase the bundled size, we will consider doing it.

**Comment by AdielCohen** [comment]
> Well - to be fair - Safari 13 is over 4 years old already and, according to the [caniuse usage-table](https://caniuse.com/usage-table), has a 0.07% market share.
> 
> Officially, the [browser support for the Maps JavaScript API](https://developers.google.com/maps/documentation/javascript/browsersupport) only covers the last two major versions of all browsers (in case of Safari that's versions 16 and 17), although in reality it's usually more than that, but no guarantees given.
> 
> Without knowing what it is that Safari 13 has a problem with, there's not really something I can recommend. Do you know if the Maps JavaScript API in a current version still works there? How about a current React version?
> 
> If both are the case, there might be a way to build a version of this library that also works there. We don't have anything too exotic going on in our codebase, so there could be some set of polyfills or compiler settings to enable support.
> 
> And while that's not something we will put on our roadmap, feel free to give it a try. And if there's a simple thing we can do that doesn't slow down the library for modern browsers or increase the bundled size, we will consider doing it.

I use the last verstion of react, vite, and react-google-maps. I don't have any error but I still can't see the map.
Anyway, I want to thank you very much for the detailed answer, I really appreciate the help.
I hope my users will use the last version of browsers.

**Comment by usefulthink** [comment]
I guess the important point here is this: If it's the Maps API itself that's no longer working in that browser, there's nothing we can do about it. But if it works, there might be things to adjust in our build to make it work. And while I personally don't feel the need to investigate this further, if there's an easy thing we can change to add that support I wouldn't be opposed.

**Comment by AdielCohen** [comment]
@usefulthink 
Is there build-in option to show message "Your browser not supported"?

**Comment by usefulthink** [comment]
No, unfortunately not. At runtime, we can't really know which parts older browsers might have a problem with and it wouldn't really be economical to implement feature-tests for every js- and browser feature that ends up being used in the library.

**Comment by stuyy** [comment]
I just tried this library out, I am on latest versions of Firefox, Edge, Chrome, and Brave, and it does not work/show up on any of the browsers.
